pages/api/feed/posts.tsx

```diff
@@ -33,17 +30,22 @@ const filterAndFormatPost = (data) => {
   return result;
 };
 
+const _getPost = async (offset = 0, limit = 100) => {
+  try {
+    const { data } = await axios.get(`${FCD_URL}v1/txs?offset=${offset}&limit=${limit}&account=${ADDRESS}`);
+    return { ...data };
+  } catch (err) {
+    return { err };
+  }
+};
+
 export const getPost = async (offset = 0, limit = 100) => {
   try {
-    const query = `${CLUB_SERVER_ROOT}/posts?offset=${offset}&limit=${limit}`;
-    const postRequest = await axios.get(query, {
-      headers: {
-        'X-Requested-With': 'XMLHttpRequest',
-      },
-    });
+    const result = await _getPost(offset, limit);
+    const postRequest = result ?? [];
```

Comment:
We don't want to create another api, please create a hook in the data folder and move the getPost function there.